src/hooks/web3Context.tsx

```diff
@@ -2,22 +2,23 @@ import React, { useState, ReactElement, useContext, useEffect, useMemo, useCallb
 import Web3Modal from "web3modal";
 import { StaticJsonRpcProvider, JsonRpcProvider, Web3Provider } from "@ethersproject/providers";
 import WalletConnectProvider from "@walletconnect/web3-provider";
+import { EnvHelper } from "../helpers/Environment";
 
 // NOTE(zx): Want to move away from infura. Will probably remove these.
 var INFURA_ID_LIST: any[];
-if (process.env.REACT_APP_INFURA_IDS) {
-  INFURA_ID_LIST = process.env.REACT_APP_INFURA_IDS.split(" ");
+if (EnvHelper.env.REACT_APP_INFURA_IDS) {
+  INFURA_ID_LIST = EnvHelper.env.REACT_APP_INFURA_IDS.split(" ");
```

Comment:
@lolchocotaco it just occurred to me that the `EnvHelper` I created is not exactly what you want.
Are you wanting me to encapsulate all of these if statements (like the above, referenced) into logic in the helper?

That way we don't have `if (EnvHelper.env...` & we can simply have `INFURA_ID_LIST = EnvHelper.INFURA_ID_LIST`